src/tests/ftest/launch.py

```diff
@@ -636,22 +661,31 @@ def get_nvme_replacement(args):
 
     # Get the list of NVMe PCI addresses found in the output
     output_str = "\n".join([line.decode("utf-8") for line in output_data[0][0]])
-    devices = find_pci_address(output_str)
+    if include_vmd_flag == "vmd" or include_vmd_flag == "mixed":
```

Comment:
(pylint-consider-using-in) Consider merging these comparisons with "in" to "include_vmd_flag in ('vmd', 'mixed')"